spec/amp-framework-hosting.md

```diff
@@ -66,7 +75,7 @@ The built framework can be found in directory `dist`. The version assigned to th
 
 If you have advanced hosting capabilities or would like to manually assign a version, `gulp dist` accepts these flags (among others):
 
-- `--config`: Indicate the release type, production (`prod`) or canary (`canary`). Defaults to `prod`.
+- `--config`: Indicate the release type, stable (`prod`) or experimental (`canary`). Defaults to `prod`.
```

Comment:
This is the one case where "production" and "canary" are the correct terms - these refer to which config file is prepended to v0.js (and other base runtimes such as amp4ads-v0.js) and those are referred to as production and canary

Reply:
Done